platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Hash.java

```diff
@@ -26,21 +26,23 @@
 import com.swirlds.common.io.streams.AugmentedDataOutputStream;
 import com.swirlds.common.io.streams.SerializableDataInputStream;
 import com.swirlds.common.io.streams.SerializableDataOutputStream;
+import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 import java.io.IOException;
 import java.io.Serializable;
 import java.nio.ByteBuffer;
 import java.util.Arrays;
+import java.util.Objects;
 
 /**
  * A cryptographic hash of some data.
  */
 public class Hash implements Comparable<Hash>, SerializableWithKnownLength, Serializable {
-    private static final int SHORT_STRING_BYTES = 4;
     public static final long CLASS_ID = 0xf422da83a251741eL;
     public static final int CLASS_VERSION = 1;
 
     private byte[] value;
+    private Bytes bytes;
```

Comment:
Are we storing the same bytes twice?

Reply:
No, the second field is a `com.hedera.pbj.runtime.io.buffer.Bytes` which is a tiny thing that holds a ~pointer~reference to the `byte[] value` + offset + length.  So it's sharing the `value` array.

Reply:
The idea is that `value` will be removed as soon as possible, but it was too much to put into 1 PR